driver-core/src/test/java/com/datastax/driver/core/TestListener.java

```diff
@@ -28,7 +29,13 @@
     private int test_index = 0;
 
     static {
-        System.out.println("[CCMBridge] Using Cassandra version: " + CCMBridge.CASSANDRA_VERSION + " (install arguments: " + CCMBridge.CASSANDRA_INSTALL_ARGS + ")");
+        if (CCMBridge.isDSE()) {
+            System.out.println("[CCMBridge] Using DSE version: " + CCMBridge.getDSEVersion() + " (C* " +
+                    CCMBridge.getCassandraVersion() + ", install arguments: " + CCMBridge.getInstallArguments() + ")");
+        } else {
+            System.out.println("[CCMBridge] Using Cassandra version: " + CCMBridge.getCassandraVersion() +
```

Comment:
Minor nit: I always considered this message a bit misleading. It makes believe that a CCM cluster is being launched at that moment, which is not true; besides, this message is printed even when running only unit tests. Maybe we could rephrase it "[CCMBridge] CCM tests for this test session will use ...."